student_class.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
```

Comment:
- [ ] The name has a default value `"Unknown"` and the parent_permission has a default value `true`, kindly fix that 🙏🏾